### PR TITLE
ClusterLoader - Adding measurement String() method

### PR DESCRIPTION
**Pull request by krzysied**
Adding `String()` method to measurement interface.

ref #275

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/simple/wait_for_pods.go [hunk 1]
```diff
@@ -99,6 +104,7 @@ func waitForPods(clientSet clientset.Interface, namespace, labelSelector, fieldS
 	defer ps.Stop()
 
 	var podsStatus measurementutil.PodsStartupStatus
+	selectorsStrnig := createSelectorsString(namespace, labelSelector, fieldSelector)
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/simple/wait_for_pods.go]: String

File: clusterloader2/pkg/measurement/common/simple/wait_for_pods.go [hunk 2]
```diff
@@ -140,3 +146,20 @@ func waitForPods(clientSet clientset.Interface, namespace, labelSelector, fieldS
...
+		selectorsStrings = append(selectorsStrings, fmt.Sprintf("namespace(%s)", namespace))
+	}
+	if labelSelector != "" {
+		selectorsStrings = append(selectorsStrings, fmt.Sprintf("labelSelector(%s)", labelSelector))
+	}
+	if fieldSelector != "" {
+		selectorsStrings = append(selectorsStrings, fmt.Sprintf("fieldSelector(%s)", fieldSelector))
+	}
+	if len(selectorsStrings) == 0 {
+		return "all pods"
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/simple/wait_for_pods.go]: "Everything"

File: clusterloader2/pkg/measurement/common/simple/wait_for_pods.go (hunk 1 above)
Comment (krzysied) [reply clusterloader2/pkg/measurement/common/simple/wait_for_pods.go]: Fixed.

File: clusterloader2/pkg/measurement/common/simple/wait_for_pods.go (hunk 2 above)
Comment (krzysied) [reply clusterloader2/pkg/measurement/common/simple/wait_for_pods.go]: Done.

**Pull request merged**